resources/scripts/api/server/types.d.ts

```diff
@@ -21,3 +21,13 @@ export interface ServerEggVariable {
     isEditable: boolean;
     rules: string[];
 }
+
+export interface ServerLog {
+    uuid: string;
+    user: string;
+    action: string;
+    device: string[];
+    metadata: string[];
```

Comment:
why is metadata an array, surely this is an object - in which case the type would be Record<string, any>